### ISSUE TITLE
bug : CdvPurchase.store.validator is not defined

### ISSUE THREAD

**Issue by gxolin** [j3k0/cordova-plugin-purchase#1612]
### Observed behavior

[The CdvPurchase.store.validator](https://github.com/j3k0/cordova-plugin-purchase/blob/master/api/classes/CdvPurchase.Store.md#validator) exposes a `store.validator` property, but this property is not defined in the [store final build](https://github.com/j3k0/cordova-plugin-purchase/blob/master/www/store.js)

Trying to call it will throw errors

### Expected behavior
No inaccessible properties defined in store.ts src but not accessible in store.js build

### System Info
package.json : 
```
    "@awesome-cordova-plugins/core": "^6.10.0",
    "@awesome-cordova-plugins/in-app-purchase-3": "^6.10.0",
    "cordova-plugin-purchase": "^13.11.1",
```

Output of `cordova info`.
`cordova: command not found`

**Comment by Dexus** [comment]
Then just use it properly! https://github.com/j3k0/cordova-plugin-purchase/blob/master/www/store.js#L100-L110

**Comment by gxolin** [comment]
Thanks for the advice, but my setup is based on a workaround due to #1610, and I don't use Iaptic as a backend, I have my own. I should have put the context in my issue, sorry:
- Cordova is not installed because it's a angular+capacitor project.
- I can use any other functions from the store.ts without any issue.
- \@awesome-cordova-plugins/in-app-purchase-3 is currently not available on npm but it's a wrapper I made around cordova-plugin-purchase to work around #1610.  
- the error printed is a global catch from the \@awesome-cordova-plugins when a method is not defined in the library. store.ts has this validator function, but it's undefined in the real store.js. It's probably set up later by the library, but only when installed directly, not when wrapped by \@awesome-cordova-plugins

**Comment by gxolin** [comment]
yes, using the `cordova-plugin-purchase` through the `@awesome-cordova-plugins/in-app-purchase-3` is working fine beside the validator function, I've tested the whole subscription purchase with both Android and iOS stores, and live subscription until payment, so I think everything is well configured

**Comment by Dexus** [comment]
I use it also in a CapacitorJS Projekt, but don't have the problems. So I think its something wrong with your setup. Did you import the cordova.js in your html file?

**Comment by gxolin** [comment]
I don't think it's required to install cordova, but I'm new to capacitor so I'm maybe wrong.
Here is the result of npx cap ls : 
```
> npx cap ls
[info] Found 1 Cordova plugin for android:
       cordova-plugin-purchase@13.11.1
[info] Found 1 Cordova plugin for ios:
       cordova-plugin-purchase@13.11.1
[info] Listing plugins for web is not possible.
```
but again, I don't think it's an issue here since my plugin is discovered and usable on both Android and iOs, only the store.validator function is not working properly

**Comment by Dexus** [comment]
I have test it without the awesome cordova plugins, and I cant see any error. So it looks like its a problem of your setup.
